pkg/kubelet/dockershim/docker_service.go

```diff
@@ -119,10 +119,13 @@ type NetworkPluginSettings struct {
 	// the plugin with PluginName may be found. The admin is responsible for
 	// provisioning these binaries before-hand.
 	PluginBinDirs []string
-	// PluginConfDir is the directory in which the admin places a CNI conf.
-	// Depending on the plugin, this may be an optional field, eg: kubenet
-	// generates its own plugin conf.
-	PluginConfDir string
+	// PluginBinDirsString is a list of directiores delimited by commas, in
+	// which the CNI configurations may be found.
+	PluginConfDirString string
```

Comment:
do we need to store both fields in NetworkPluginSettings; wouldn't PluginConfDirs be sufficient?
see [1].